lib/account.js

```diff
@@ -1,12 +1,29 @@
 module.exports = account
 
 var addTokenToUserDoc = require('./utils/add-token-to-user-doc')
+<<<<<<< HEAD
+var isValidTokenType = require('./utils/is-valid-token-type')
+var errors = require('./utils/errors')
+=======
+var errors = require('./utils/errors')
 var findUserDoc = require('./utils/find-user-doc-by-username-or-id-or-token')
+>>>>>>> 0f0e8b3462cec11dfc4661ba1d06283f35098533
```

Comment:
Yes, I found it! But this is the first time that I experienced. I think it takes some time to solve it!. Just give me some time.